source/extensions/filters/http/jwt_authn/BUILD

```diff
@@ -71,3 +69,51 @@ envoy_cc_library(
         "//source/extensions/filters/http/common:factory_base_lib",
     ],
 )
+
+envoy_cc_library(
+    name = "matchers_lib",
+    srcs = [
+        "matcher.cc",
+    ],
+    hdrs = [
+        "matcher.h",
+    ],
+    deps = [
+        ":verifier_lib",
+        "//source/common/http:header_utility_lib",
+        "//source/common/router:config_lib",
+    ],
+)
+
+envoy_cc_library(
+    name = "verifier_lib",
+    srcs = ["verifier.cc"],
+    hdrs = ["verifier.h"],
+    deps = [
+        ":verify_context_lib",
+        "@envoy_api//envoy/config/filter/http/jwt_authn/v2alpha:jwt_authn_cc",
+    ],
+)
+
+envoy_cc_library(
+    name = "verify_context_lib",
+    srcs = ["verify_context.cc"],
+    hdrs = [
+        "response_data.h",
+        "verifier_callbacks.h",
+        "verify_context.h",
```

Comment:
can we combine them into one header file?

Reply:
moved.